### PR TITLE
Feature/commons 3.2

### PR DESCRIPTION
**Pull request by major13ua**
* Migration to new LEP engine (xm_commons_version=3.2.2)
* LepManager -> LepManagementService 
* DomainEventAggregateTypeResolver  changed

### CODE REVIEW

File: src/main/java/com/icthh/xm/ms/timeline/lep/keyresolver/DomainEventAggregateTypeResolver.java [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
-public class DomainEventAggregateTypeResolver extends AppendLepKeyResolver {
+public class DomainEventAggregateTypeResolver implements LepKeyResolver {
 
     @Override
-    protected String[] getAppendSegments(SeparatorSegmentedLepKey baseKey, LepMethod method, LepManagerService managerService) {
-        DomainEvent domainEvent = getRequiredParam(method, "domainEvent", DomainEvent.class);
-        String translateToLepConvention = translateToLepConvention(domainEvent.getAggregateType());
-        return new String[]{translateToLepConvention};
+    public List<String> segments(LepMethod method) {
+        return Arrays.stream(method.getMethodArgValues())
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/ms/timeline/lep/keyresolver/DomainEventAggregateTypeResolver.java:17]: O_o just get argument by name

Comment (sergeysenja1992) [reply src/main/java/com/icthh/xm/ms/timeline/lep/keyresolver/DomainEventAggregateTypeResolver.java:17]: `method.getParameter("domainEvent", DomainEvent.class)`

File: src/main/java/com/icthh/xm/ms/timeline/lep/keyresolver/DomainEventAggregateTypeResolver.java [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
+            .filter(arg -> arg instanceof DomainEvent)
+            .map(arg -> ((DomainEvent) arg).getAggregateType())
+            .collect(Collectors.toList());
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/ms/timeline/lep/keyresolver/DomainEventAggregateTypeResolver.java:20]: this is java 17 use just `.toList()`

File: src/main/java/com/icthh/xm/ms/timeline/service/kafka/SystemQueueConsumer.java [hunk 3]
```diff
@@ -48,33 +41,17 @@ public void consumeEvent(ConsumerRecord<String, String> message) {
...
+                try (var ctx = lepManagementService.beginThreadContext()) {
+                    systemQueueProcessorService.handleSystemEvent(event);
+                }
             });
+        } catch (Exception e) {
+            log.error(e.getMessage(), e);
         }
 
-        String newRid = MdcUtils.getRid()
-            + ":" + StringUtils.defaultIfBlank(login, "")
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/ms/timeline/service/kafka/SystemQueueConsumer.java:69]: rid logic with putting login to rid was removed? why?

**Review by amedvedchuk — APPROVED** [review]

**Pull request merged**